### DISCUSSION TITLE
Set max tooltip width

### DISCUSSION THREAD

**Discussion by DonCamillo11** [livewire/flux#823] [Feature Requests] [answered]
It would be nice if you could specify the maximum width of a tooltip. If there is a lot of text, the tooltips extend across the entire page width.
![Bildschirmfoto 2024-12-06 um 16 28 29](https://github.com/user-attachments/assets/8443eb19-3e3f-4e89-8433-cae8c0eafa5a)

**Comment by igmltd** [comment, marked as answer]
You can do the below if that helps

```
 <flux:tooltip
            class="[&>[data-flux-tooltip-content]]:max-w-sm"
            content="lorum ipsum Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae">
            <flux:button icon="cog-6-tooth" icon-variant="outline" />
  </flux:tooltip>
```

**Comment by DonCamillo11** [comment]
@igmltd thanks, this is working for now.
